pre-commit.sh

```diff
@@ -1,11 +1,5 @@
 #!/usr/bin/env bash
 
-docker build -t event-recorder-image .
-echo "Created container: $(docker create -t --name event-recorder-container event-recorder-image)"
-echo "Started container: $(docker start event-recorder-container)"
-
-docker cp . event-recorder-container:/event-recorder
```

Comment:
Please add a condition to check if a docker is present and running. If the docker is not running, display a friendly message to a user to ask him/her to bring up Docker.

Reply:
I don't think this is related to this PR (in this PR we are really refactoring to make it so that you can run individual sections), but it's a nice idea.

I'll raise a new PR for that later today.